Dockerfile

```diff
@@ -6,19 +6,21 @@ FROM paritytech/ci-unified:bullseye-1.70.0 as builder
 WORKDIR /build
 
 ARG FEATURES=default
+ARG BINARY=kilt-parachain
 
 COPY . .
 
-RUN cargo build --locked --release --features $FEATURES
+RUN cargo build --locked --release --features $FEATURES -p $BINARY
```

Comment:
FYI the reason to compile all binaries in the first stage was so that we only need to run the first stage once and only compile dependencies once. this would lead to a slightly longer first stage, but you only need to run it once.

Reply:
Where do we make use of that information? I only see a second stage being used, and we don't, for instance, use the same first stage to also build the mashnet-node. Am I wrong?

Reply:
We (at least try to) do that here: https://github.com/KILTprotocol/kilt-node/blob/develop/.maintain/build-image.sh
I would need to verify that this works. The last pipeline took ~80min

Reply:
Also this PR breaks this script.

Reply:
The first stage takes around 70 min anyway. But good to know. If that is the case, I will revert this commit before merging the DIP branch in `develop`.